docs/deploy.md

```diff
@@ -143,18 +128,26 @@ Currently, we test the following two container registries:
           insecure_disable_tls_validation: <false/true> #defaults false
         ```
 
+1. Generate an "Internal Installation Values" file:
+
+   >  **NOTE:** The script requires the [BOSH CLI](https://bosh.io/docs/cli-v2-install/#install) in installed on your machine. The BOSH CLI is an handy tool to generate self signed certs and passwords.
+
+   ```console
+   ./hack/generate-internal-values.sh --values-file ${VALUES_DIR}/cf-values.yml > ${VALUES_DIR}/cf-internal-values.yml
+   ```
+
 1. Run the following commands to install Cloud Foundry on your Kubernetes cluster:
 
       i. Render the final K8s template to raw K8s configuration
 
          ```console
-         ytt -f config -f ${TMP_DIR}/cf-values.yml > ${TMP_DIR}/cf-for-k8s-rendered.yml
+         ytt -f config -f ${VALUES_DIR}/cf-values.yml -f ${VALUES_DIR}/cf-internal-values.yml > ${VALUES_DIR}/cf-for-k8s-rendered.yml
```

Comment:
the fact that we save the rendered k8s yaml here means that maybe we should keep the variable as TMP_DIR? (or use both a values dir and a tmp dir)